dovecot/install.sh

```diff
@@ -100,6 +100,9 @@ protocol lda {
 } 	
 " >> /etc/dovecot/dovecot.conf
 
+# Kill all processes (Apache) listening on port 80 because this may prevent the start of NGINX
+fuser -k 80/tcp
```

Comment:
Dovecot for some reason installs Apache2 which occupies port 80. Also, port 80 is required by NGINX. I will check why Dovecot installs Apache and I will be back.